### PR TITLE
PICARD-1220: Add keyboard shortcut to delete scripts in options page.

### PR DESCRIPTION
**Pull request by vishichoudhary**
# Summary

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

JIRA [PICARD-1220](https://tickets.metabrainz.org/browse/PICARD-1220)
# Solution
In options > Scripting. On pressing delete key, the script will be removed.

### CODE REVIEW

**Comment by sambhav** [comment]
This is a destructive command. You'd probably want some sort of confirmation.

**Comment by vishichoudhary** [comment]
@sambhav  there exist confirmation, it simply call remove method.

File: picard/ui/options/scripting.py [hunk 1]
```diff
@@ -232,6 +232,14 @@ def __init__(self, parent=None):
         self.last_selected_script_pos = 0
         self.ui.splitter.setStretchFactor(0, 1)
         self.ui.splitter.setStretchFactor(1, 2)
+        self.shortcut = QtWidgets.QShortcut(self.ui.script_list)
+        self.shortcut.setKey(QtGui.QKeySequence.Delete)
+        self.shortcut.activated.connect(self.processor_delete_item)
+
+    def processor_delete_item(self):
```
Comment (sambhav) [inline picard/ui/options/scripting.py]: Please conform to python method naming used in the rest of the project code.

Comment (vishichoudhary) [reply picard/ui/options/scripting.py]: @sambhav ```script_delete_processor``` is ok?

**Comment by sambhav** [comment]
@vishalchoudhary please put more efforts into the quality of code you produce. It gets very tiresome for both you and the reviewers to point such trivial mistakes again and again.

Please make sure that you are following the contribution guidelines WRT naming, comments, commit messages and PR descriptions.

Since this is a feature addition, I'd expect some sort of ticket to reflect the same in the changelog.

**Comment by vishichoudhary** [comment]
@sambhav ok I will open a ticket. apologies for the inconvenience.

File: picard/ui/options/scripting.py (hunk 1 above)
Comment (sambhav) [reply picard/ui/options/scripting.py]: `delete_selected_script` should do.

**Pull request closed without merging**